src/index.ts

```diff
@@ -63,9 +63,9 @@ export {
   calculateStaticTreeDepth,
   getActiveLeafRoute,
 } from './Api/NavigationUtils'
+export { useAndroidBackForCustomOnBack } from './Hooks/UseAndroidBackForCustomOnBack'
 export { useIsInitial } from './Hooks/UseIsInitial'
 export {
-  type Navigation,
```

Comment:
this is breaking change, is this necessary?  Also just noticed that we have many comments in this file, are they still needed ?

Reply:
I've moved the `Navigation` type to `src/Model/Types` because I reused it. It's already exported on line 19. I will remove the comments, they are not needed anymore.